src/deployments.ts

```diff
@@ -110,6 +110,7 @@ export const RENFT_CONTRACT_DEPLOYMENTS: RenftContractDeployments = [
   DEPLOYMENT_RESOLVER_AVALANCHE_FUJI_TESTNET_V0,
   DEPLOYMENT_RESOLVER_AVALANCHE_MAINNET_V0,
   DEPLOYMENT_RESOLVER_POLYGON_MAINNET_V1,
+  DEPLOYMENT_SYLVESTER_POLYGON_MAINNET_V1,
```

Comment:
🤦‍♀️

Thinking it would probably make sense for all deployment instances to be declared inline in this array, instead of exposing these manually. This would force callers to use the correct deployment instance using `findDeployments`.

Reply:
done.

Reply:
https://github.com/re-nft/sdk/pull/9#discussion_r1121742555